src/components/Create/components/JuiceVideoOrImgPreview.tsx

```diff
@@ -22,11 +23,13 @@ export function JuiceVideoOrImgPreview({
   const { data: contentType } = useContentType(src)
   const isVideo = fileTypeIsVideo(contentType)
 
+  const _onClose = stopPropagation(onClose)
```

Comment:
**question** why the need for stopPropagation?

Reply:
Because in the instances of `<JuiceVideoThumbnailOrImage>` in the upload forms, closing this preview triggers a click on the upload field.

Reply:
aids